docs/embedded/tutorials/register-ct.md

```diff
@@ -1,13 +1,62 @@
 ---
-title: Register Container Types
+title: Create and Register Container Types
 description: This article explains how to register a new Container Types work.
 ms.date: 11/28/2023
 ms.localizationpriority: high
 ---
 
-# Registering a Container Type
+# Create and Register a Container Type
 
-## Create a new Container Type for your app
+This article will explain how to create and register a billable [Container Type](../concepts/app-concepts/containertypes.md) in SharePoint Embedded.
+
+**Pre-requisites:**
+* You must have admin credentials to a M365 developer tenant
+* You must register your application in [Microsoft Entra ID](https://learn.microsoft.com/en-us/training/modules/sharepoint-embedded-setup/5-exercise-setup-configure-sharepoint-embedded)
+
+
+
+## Set up an Azure Subscription Id, Resource Group, and Region
+
+### 1. Set up Azure Subscription Id
+Login to Azure Portal with the admin credentials of your tenant
+
+
+(TODO: add the extra existing Sub id)
+
+Navigate to [Subscriptions](https://portal.azure.com/#view/Microsoft_Azure_Billing/SubscriptionsBladeV2)
+![alt text](azuresubscription.png)
+
+If you do not have an existing subscription set-up, click add
+![alt text](suboption.png)
+
+Choose a subscription plan. You will be prompted to add in credit card information that will be billed when you create File Storage Containers. This will set you up with an Azure Subscription Id.
+
+### 2. Set up a Resource Group and Region
+Navigate to [Resource Groups](https://portal.azure.com/#view/HubsExtension/BrowseResourceGroups) in Azure Portal.
+![alt text](resource-group.png)
+
+Click "Create" to create a resource group and name it something appropiate to your application. Then click Review + Create
+![alt text](c-rg.png)
+
+Make sure that:
+* The appropiate Azure Subcription is selected for the resource group
+* The appropiate physical region is selected. For ex. (US) East US
+
+>> Note: The Region Group does NOT require your Containers to live in that specified region. Your content is not restricted to the specified region
+
+**TODO: Need to explain Region and Resource Group in relation to SharePoint Embedded
+**
+
+Click Review + Create and your validation should pass if the inputs are correct.
+
+Now you have all the necessary resources to create a Billable Container Type.
+
+<br></br>
```

Comment:
don't use html in markdown unless aboslutely necessary (like breaklines within tables). these should be removed